tbx/people/models.py

```diff
@@ -79,14 +87,35 @@ def author_posts(self):
 
     @cached_property
     def related_works(self):
-        # Get the latest 2 work pages by this author
-        works = (
-            HistoricalWorkPage.objects.filter(authors__author__person_page=self.pk)
+        """Returns work pages authored by the person, giving preference to Work rather than Historical work pages"""
+        # Get the latest 3 work pages by this author
+        recent_works = (
+            WorkPage.objects.filter(authors__author__person_page=self.pk)
             .live()
             .public()
             .distinct()
-            .order_by("-date")[:2]
+            .order_by("-date")[:3]
         )
+
+        remaining_slots = 3 - len(recent_works)
+
+        if remaining_slots == 0:
+            # No historical works needed, just return recent works
+            works = recent_works
+
+        else:
+            # Get the latest 3 historical work pages by this author iff necessary
+            historical_works = (
+                HistoricalWorkPage.objects.filter(authors__author__person_page=self.pk)
+                .live()
+                .public()
+                .distinct()
+                .order_by("-date")[:remaining_slots]
+            )
+
+            # Combine the two querysets and get the first three results
+            works = list(chain(historical_works, recent_works))
```

Comment:
In the template I am seeing more than 3 work posts - an example is Edd - http://localhost:8000/team/edd-baldry/

Reply:
Ah yes, I've just updated this now